modules/order/src/OrderRefresh.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Drupal\commerce_order;
+
+use Drupal\commerce_order\Entity\OrderInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+
+/**
+ * Default implementation for order refresh.
```

Comment:
I tend to leave out docblocks that don't add anything of value nowadays (like the "default implementation" one). Keep it if you disagree.